extensions/kubectl-cli/package.json

```diff
@@ -147,6 +147,7 @@
   "dependencies": {
     "@podman-desktop/api": "^0.0.1",
     "@octokit/rest": "^21.0.1",
+    "@types/semver": "^7.5.8",
```

Comment:
`@types/*` should be in devDependencies because they are used for development, they provide type information for the typescript compiler. You should also add the `semver` to the `dependencies` because it is used at runtime.